api/src/main/java/graphql/nadel/Nadel.java

```diff
@@ -70,6 +70,7 @@
     final ServiceExecutionFactory serviceExecutionFactory;
     final List<Service> services;
     final GraphQLSchema overallSchema;
+    @NotNull
```

Comment:
Interesting. Does placing the annotation here has the same effect of placing it in the constructor?

Reply:
Uh, tbh idk I only put it there because I used it from Kotlin code. But then I deleted that code. So I'm just leaving this in.